### PR TITLE
Use Factories -> \Helpers\URLHelperTest.php & \HTTP\URITest.php

### PR DESCRIPTION
**Pull request by totoprayogo1916**
**Description**
- remove `helper('text')` , which is already running on the `mb_url_title()`
- Use Factories

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: tests/system/Helpers/URLHelperTest.php [hunk 1]
```diff
@@ -1278,8 +1279,6 @@ public function testUrlTitleExtraDashes()
 
 	public function testMbUrlTitle()
 	{
-		helper('text');
```
Comment (MGatner) [inline tests/system/Helpers/URLHelperTest.php:1281]: Why were these removed?

File: tests/system/Helpers/URLHelperTest.php [hunk 2]
```diff
@@ -1295,8 +1294,6 @@ public function testMbUrlTitle()
 
 	public function testMbUrlTitleExtraDashes()
 	{
-		helper('text');
```
Comment (MGatner) [inline tests/system/Helpers/URLHelperTest.php:1298]: Why were these removed?

**Review by MGatner — CHANGES_REQUESTED** [review]

File: tests/system/Helpers/URLHelperTest.php (hunk 1 above)
Comment (MGatner) [reply tests/system/Helpers/URLHelperTest.php:1281]: Oh nevermind I just read your comment! Good catch.

**Review by MGatner — APPROVED** [review]

**Comment by MGatner** [comment]
I'm not sure what is happening with the tests but I don't think it is related to the PR content. Rerunning just to check.

**Comment by totoprayogo1916** [comment]
The tests still error but
on my fork, the test passed.

**Comment by paulbalandan** [comment]
Please rebase.

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**